File: lib/python.lib/tsunagi.py
import socket
import time

HOST = '127.0.0.1'  
PORT = 5000         

with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
    s.connect((HOST, PORT))

    client_id = "Murasaki"
    s.sendall(client_id.encode())
    print(f"Nome enviado: {client_id}\n")

    time.sleep(1)

    messages_plan = [
        ("2", 5),  
        ("1", 2), 
        ("5", 3),  
        ("7", 4), 
    ]

    count = 1
    try:
        for msg_id, qty in messages_plan:
            for _ in range(qty):
                payload = f"Mensagem {count}"
                mensagem_com_id = f"{msg_id}|{payload}"
                s.sendall(mensagem_com_id.encode())
                print(f"Enviado: {mensagem_com_id}")
                count += 1
                time.sleep(0.05)  
    except KeyboardInterrupt:
        print("\nCliente encerrado pelo usuário.")

File: lib/javascript.lib/tsunagi.js
const net = require("net");

const HOST = "127.0.0.1";
const PORT = 5000;

const client = new net.Socket();

client.connect(PORT, HOST, () => {
  console.log("Conectado ao servidor");

  const clientId = "Gym";
  client.write(clientId);
  console.log(`Nome enviado: ${clientId}\n`);

  const messagesPlan = [
    ["2", 5],
    ["1", 2],
    ["5", 3],
    ["7", 4],
  ];

  let count = 1;

  const sendMessages = async () => {
    for (const [msgId, qty] of messagesPlan) {
      for (let i = 0; i < qty; i++) {
        const payload = `Mensagem ${count}`;
        const mensagemComId = `${msgId}|${payload}`;
        client.write(mensagemComId);
        console.log(`Enviado: ${mensagemComId}`);
        count++;
        await new Promise((resolve) => setTimeout(resolve, 3000));
      }
    }
  };

  sendMessages().catch(console.error);
});

client.on("data", (data) => {
  console.log("Recebido do servidor:", data.toString());
});

client.on("close", () => {
  console.log("Conexão encerrada");
});

client.on("error", (err) => {
  console.error("Erro:", err.message);
});
